backend/enums/notifications.js

```diff
@@ -41,47 +46,59 @@ export class Notifications {
             from: details.userId,
             post: details.postId,
             timestamp: Date.parse(details.createdAt),
-            content: null
+            content: null,
         }
     }
 
     static async retrieveAllReplies(userPosts) {
-        let allReplies = [];
-        for(const post of userPosts){
-            let replies = await post.getReplies(post.id)
-            replies.map(r => allReplies.push(this.convertToNotifications(Notifications.reply, r)))
+        const allReplies = []
+        for (const post of userPosts) {
+            const replies = await post.getReplies(post.id)
```

Comment:
Refactor this await in loop to use `Promise.all` in order to remove the eslint warning.

Reply:
Yes that was a good idea